### PR TITLE
Add @iNewLegend to CODEOWNERS ED-2651

### PR DESCRIPTION
**Pull request by iNewLegend**

### CODE REVIEW

**Review by kobizz — CHANGES_REQUESTED** [review]
Don't override existing rules

**Comment by iNewLegend** [comment]
> Don't override existing rules
@kobizz 
please explain

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,104 +1,127 @@
 # Default
-*                                       @kingyes
-*.js                                    @kobizz @matipojo
-*.php                                   @matipojo @kobizz
-*.json                                  @kingyes
-*.xml                                   @kingyes
+# To stay with fixed indent use tabs allways.
+*								@kingyes
```
Comment (matipojo) [inline .github/CODEOWNERS:3]: Try to use spaces instead of tabs.

**Review by matipojo — CHANGES_REQUESTED** [review]

File: .github/CODEOWNERS (hunk 1 above)
Comment (iNewLegend) [reply .github/CODEOWNERS]: Handled.

**Review by matipojo — APPROVED** [review]

**Pull request merged**